README.md

```diff
@@ -50,6 +46,48 @@ by Jenkins, this will alert developers.
 
 `bundle exec rspec`
 
+### Testing the alert checks
```

Comment:
Could we remove this section?

Reasons:

- Parts of it are external to this repo e.g. "2 days old" and "Icinga".
- Parts of it are incomplete e.g. "Jenkins job" <- what Jenkins job?!

Since we have [a dev doc](https://docs.publishing.service.gov.uk/manual/alerts/email-alerts-travel-medical.html) for this check, I think it's safe to delete this somewhat-harder-to-find version of it.

Reply:
Hmm I'd like to keep this section as I think it serves a separate purpose to the dev doc. The dev doc is about trouble shooting, this guidance is about testing the changes you've made to the check are working as expected.

As this is an important part of our alerting system then documenting how to test changes to the repo seems useful. 

I have taken on board your comments though and thrown in some links and code references which I think makes the document clearer and more reliable.

Thoughts?

Reply:
Fair enough - it's much more usable than it was, too, with those links.